app/main/db/queries.py

```diff
@@ -560,3 +560,46 @@ def get_all_transferring_bodies():
     for body in Body.query.all():
         bodies.append(body.Name)
     return bodies
+
+
+def get_breadcrumb_values(
```

Comment:
can we just split this out into 3 different functions, 1 for body, 1 for series, 1 for consignment since there is no shared logic for any of these?

We will also need one for the record page to be consistent.